.rubocop.yml

```diff
@@ -1,3 +1,7 @@
+AllCops:
+  Exclude:
+    - '**/Rakefile'
+    - '*.gemspec'
```

Comment:
I'd rather not ignore all cops for these files, can we work through the issues together?

Reply:
You're right, I probably went overboard there thinking "It's more DSL than Ruby, so checks don't make sense at all." I'll make the exclude more specific.